mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -708,16 +708,18 @@
     <!-- Text for the dialog to confirm clearing all downloads -->
     <string name="download_delete_all_dialog">Are you sure you want to clear your downloads?</string>
     <!-- Text for the snackbar to confirm that multiple downloads items have been deleted -->
-    <string name="download_delete_multiple_items_snackbar">Downloads Deleted</string>
+    <string name="download_delete_multiple_items_snackbar_1">Downloads Removed</string>
+    <!-- Text for the snackbar to confirm that a single download item has been deleted. The first parameter is the name of the download item. -->
```

Comment:
Same here, comment should reflect that download item has been "removed" (since not the same as "deleted")